lib/util.js

```diff
@@ -430,3 +431,150 @@ module.exports = exports = {
                   'util.puts is deprecated. Use console.log instead.',
                   'DEP0027')
 };
+
+if (process.binding('config').experimentalReport) {
+  try {
+    // if node-report is enabled, extract the binding and
+    // wrap the APIs with thin layers, with some error checks.
+    // user options can come in from CLI / ENV / API.
+    // CLI and ENV is intercepted in C++ and the API call here (JS).
+    // So sync up with both sides as appropriate - initially from
+    // C++ to JS and from JS to C++ whenever the API is called.
+    // Some events are controlled purely from JS (signal | exception)
+    // and some from C++ (fatalerror) so this syncup is essential for
+    // correct behavior and aligned to the supplied tunables.
+    const nr = internalBinding('node_report');
+
+    var report = {
+      events: [],
+      signal: 'SIGUSR2',
+      filename: '',
+      path: '',
+      verbose: false,
+
+      setDiagnosticReportOptions: function(options) {
```

Comment:
Perhaps use the shorthand notation here?
```js
  setDiagnosticReportOptions(options) {
  // ...
  },
```

Reply:
fixed.